### DISCUSSION TITLE
Allow widgets to appear from results when using finder

### DISCUSSION THREAD

**Discussion by Henzxon** [elementor/elementor#23980] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Its frustrating when you want to insert a global widget or ordinary widget and theres a lot of back and forth between the widgets panel into the visual builder.

### Describe the solution you'd like

It would be nice if widget can also be incorporated to the finder and widgets will appear based on search query.

**Comment by awakekat** [comment]
I am curious as to why you are wanting to look for widgets outside of the editor? The editor already has a search feature to find widgets.
![Screenshot 2023-11-02 at 8 07 07 AM](https://github.com/elementor/elementor/assets/5148843/9b7e9555-113a-4af4-8bdb-08688d339ad9)

**Reply by Henzxon** [reply]
Convenience and Accessibility really

- Hitting Command + E or Ctrl + E is way faster than dragging your mouse across the screen all the time.
Imagine having to drag your mouse from the right side of your display to the top left corner of your screen all the time, it might be small but that's significant over time.

- Keyboard arrow and tab navigation on results is possible for accessibility.
<img width="736" alt="image" src="https://github.com/elementor/elementor/assets/102000469/b60db27c-2a6b-4266-aecb-53f2cf9829af">

- The search finder can be dragged anywhere on the screen

- This would also make users use the finder more often when searching for certain basic or global widget

As an experiment, you might be used to dragging your mouse on the top left corner but try tracing back your mouse movement when adding 3 basic widgets.

These request are just random ideas, developers might or might not want to implement these features but certain users and power users might want these kind of features and having multiple options to achieve the same task is always good to cater a broader range of users.

Thank you
